src/game/map/CMakeLists.txt

```diff
@@ -4,12 +4,13 @@ set(SRCROOT ${SRCROOT}/map)
 add_subdirectory(cellular)
 
 set(SRC
-    ${SRC}
-    ${SRCROOT}/Map.cpp
-    ${SRCROOT}/Map.hpp
-    PARENT_SCOPE)
+	${SRC}
+	${SRCROOT}/Map.cpp
+	${SRCROOT}/Map.hpp
+	${SRCROOT}/CreateMap.cpp
+	PARENT_SCOPE)
 
 set(INCDIRS
-    ${INCDIRS}
-    ${INCROOT}
-    PARENT_SCOPE)
+	${INCDIRS}
+	${INCROOT}
+	PARENT_SCOPE)
```

Comment:
Please use 4 spaces for indentation inside CMakeList.txt files

Reply:
![image](https://user-images.githubusercontent.com/72017980/174244795-0e5ba9e7-fc12-45e7-a573-9733ef61e099.png)